routes/api/upgrade/connect.js

```diff
@@ -0,0 +1,70 @@
+var express = require('express'),
+    router = express.Router();
+
+var responseOk = {
```

Comment:
Completely minor, specially for a mocked service like this: You can use the same `var` statement in line #1 like this:
```
var express = require('express'),
    router = express.Router(),
    responseOk = {
```